### PR TITLE
MDS-2606/enable flask logging in uwsgi.

### PR DESCRIPTION
**Pull request by Jsyro**

### CODE REVIEW

**Comment by Jsyro** [comment]
SAMPLE FROM LOGS: Three lines, 1 and 3 are uWSGI request logs, Line 2 is a coded flask logger line, the formatting can be adjusted easily. 


[pid: 30\|app: 0\|req: 54/125] 172.51.80.1 () {56 vars in 5293 bytes} [Thu Feb 13 22:23:09 2020] GET /1199/api/now-applications/faa6d155-b226-4dce-aeb5-26968d002a99?original=True => generated 1909 bytes in 9103 msecs (HTTP/1.1 200) 5 headers in 152 bytes (1 switches on core 2)
--
  | [2020-02-13 22:23:19,503] INFO in now_application_resource: Assigning <NOWApplicationIdentity UUID('faa6d155-b226-4dce-aeb5-26968d002a99')> to <Mine UUID('1b6c1923-bc93-4e3b-9d0c-1dac5a984bc1')>
  | [pid: 34\|app: 0\|req: 72/127] 172.51.80.1 () {62 vars in 5402 bytes} [Thu Feb 13 22:23:13 2020] PUT /1199/api/now-applications/faa6d155-b226-4dce-aeb5-26968d002a99 => generated 3656 bytes in 12988 msecs (HTTP/1.1 200) 6 headers in 218 bytes (1 switches on core 2)

File: services/core-api/app/__init__.py [hunk 1]
```diff
@@ -28,6 +32,25 @@
 
 def create_app(test_config=None):
     """Create and configure an instance of the Flask application."""
+    dictConfig({
```
Comment (JustinMacaulay) [inline services/core-api/app/__init__.py]: Maybe this belongs in a config file?

File: services/core-api/app/commands.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import click
 import psycopg2
+#from logging import getLogger, _nameToLevel
```
Comment (JustinMacaulay) [inline services/core-api/app/commands.py]: Missed import removal?

**Comment by Jsyro** [comment]
So a config change is how we change the logging level, however.... it fired the prehook and all the migrations, so it might be worth looking at those to speed them up (mds_test db is always nuked and paved)

**Pull request merged**